dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -88,6 +97,42 @@ private static void generateReports(Configuration configuration, Path output) {
     }
   }
 
+  @VisibleForTesting
+  static List<String> readBom() throws ArtifactDescriptorException {
+    
+    DefaultArtifact artifact =
+        new DefaultArtifact("com.google.cloud:cloud-oss-bom:pom:0.0.1-SNAPSHOT");
```

Comment:
It seems like it would be a pain to update this every time the version bumps up. Have you thought about how we could manage to control the version in a single place? In google-cloud-java, we do this using a file with versions ( https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/versions.txt ), tags throughout the repo, and a version updater script ( https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/utilities/replace_versions.py ).

Reply:
That makes sense. It's likely part of figuring out how to read from the local code instead of the repo. Haven't figure that out yet.